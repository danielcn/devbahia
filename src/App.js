import React from 'react';
import ReactDOM from 'react-dom';

const App = () =>(
    <div>
        <h1>Hello Dev Bahia</h1>
    </div>   
);

export default App;