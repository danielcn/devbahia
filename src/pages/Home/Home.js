import React, {Component} from 'react';
import Card from '@material-ui/core/Card';
import CardContent from '@material-ui/core/CardContent';
import Header from '../../components/Header/Header';
import MediaCard from '../../components/MediaCard/MediaCard';

const fethedArray = [1, 2, 3, 4, 5 , 6, 7];

const mediaCards = () => fethedArray.map((index, value) => {
  return <MediaCard  key={index} />;
});

class Home extends Component {
  render () {
    return (
      <div className="Home">
        <Header/>
        <Card>
          <CardContent>
            <h1>Hello Dev Bahia</h1>
            { mediaCards() }
          </CardContent>
        </Card>
      </div>
    );
  }
}

export default Home;
