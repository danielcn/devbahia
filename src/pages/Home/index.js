import React, {Component} from 'react';

import Button from '@material-ui/core/Button';
import Card from '@material-ui/core/Card';
import CardActions from '@material-ui/core/CardActions';
import CardContent from '@material-ui/core/CardContent';

import Header from '../../components/Header';

class Home extends Component {
    render () {
        return (
            <div className="Home">
                <Header/>
                <Card>
                    <CardContent>
                        <h1>Hello Dev Bahia</h1>
                        <CardActions>
                            <Button size="small" color="primary">Learn More</Button>
                        </CardActions>
                    </CardContent>
                </Card>
            </div>
        );
    }
}

export default Home;
