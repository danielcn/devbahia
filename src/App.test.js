import React from 'react';
import { shallow } from 'enzyme';
import Home from './pages/Home/Home'
import App from './App';

describe('App Snapshot', () => {
    test('renders', () => {
      const app = shallow(<App />)
      expect(app).toMatchSnapshot();
    });

    test('should render without throwing an error', function() {
      const app = shallow(<App />);
      expect(app.contains(<Home />)).toBe(true);
    });
});
