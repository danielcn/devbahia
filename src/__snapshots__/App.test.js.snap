// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot renders 1`] = `ShallowWrapper {}`;
